commons/src/test/java/org/frankframework/core/IbisExceptionTest.java

```diff
@@ -116,14 +135,6 @@ public void sqlExceptions(String expectedMessage, SQLException cause) {
 		assertEquals(expectedMessage, result);
 	}
 
-	private static Stream<Arguments> oracleXaExceptions() {
-		return Stream.of(
-			Arguments.of("text: (OracleXAException) XAErr (0): Internal XA Error ORA-1234 SQLErr (0)", new OracleXAException(1234, 0)),
-			Arguments.of("text: (OracleXAException) xaError [5678] xaErrorMessage [Internal XA Error]: XAErr (5678): Internal XA Error ORA-1234 SQLErr (0)", new OracleXAException(1234, 5678)),
-			Arguments.of("text: (OracleXAException) xaError [1234] xaErrorMessage [Internal XA Error]: XAErr (1234): Internal XA Error ORA-1234 SQLErr (0): (SQLException) SQLState [xyz], errorCode [1234]: sql reason: rootException", new OracleXAException(new SQLException("sql reason", "xyz", 1234, new IbisException("rootException")), 1234))
-		);
-	}
-
 	@ParameterizedTest
 	@MethodSource("oracleXaExceptions")
 	public void sqlExceptions(String expectedMessage, OracleXAException cause) {
```

Comment:
Tja hier vind ik het eigenlijk niet zo gek dat de statics er tussen staan maargoed...

Reply:
Hmm, ik dacht dat ik hier op had gereageerd. Maar ik ben het met je eens, dit hoort juist bij de test

Reply:
Ik heb dit kunnen aanpassen in de settings door het item met de pijl te verwijderen (zie plaatje hieronder). Daarna opnieuw de XML bestanden opgemaakt met de formatting settings. Ik zie geen wijzigingen in Eclipse terugkomen. @nielsm5 , zou je kunnen kijken of je een vergelijkbare setting in Eclipse handmatig kunt wijzigen?
<img width="1007" alt="code_style" src="https://github.com/user-attachments/assets/39ad4797-3ec5-49ac-905e-0480e2f3d6fd">

Reply:
IntelliJ kan niet voor Eclipse een configuratie maken, doel is nu puur en alleen om IntelliJ goed te krijgen.